### ISSUE TITLE
Forbit users from creating new boards

### ISSUE THREAD

**Issue by belf88** [wekan/wekan#4475]
Hi everyone, how can I forbit the creation of boards for some user accounts?

Ty

**Comment by mfilser** [comment]
I guess not.
Also i think first #3022 should be done, then it's easier to implement this option.

**Comment by belf88** [comment]
> I guess not. Also i think first #3022 should be done, then it's easier to implement this option.

I read, but I would need to ban the creation of new boards in the main desk.

**Comment by mfilser** [comment]
@belf88
I already understood what you want. What i wanted to say is that Wekan (as i think) first needs a flexible permission role concept before implementing new ones

**Comment by belf88** [comment]
> @belf88 I already understood what you want. What i wanted to say is that Wekan (as i think) first needs a flexible permission role concept before implementing new ones

True, project is very focused on the permissions of the individual boards, and not on general system permissions. Unfortunately without this setting i cannot use wekan. I have to allow my customers to see only the boards shared with them. Customers themselves don't have to create new boards.

**Comment by xet7** [comment]
@belf88 

By default, new WeKan users do not see any boards. Only if you share some board, it will become visible to that user.

**Comment by belf88** [comment]
> @belf88
> 
> By default, new WeKan users do not see any boards. Only if you share some board, it will become visible to that user.

Ok,  but I have to ban users from creating new boards

**Comment by belf88** [comment]
> @belf88
> 
> By default, new WeKan users do not see any boards. Only if you share some board, it will become visible to that user.

Is it possible to set that only the administrator user can create new boards?

**Comment by xet7** [comment]
@belf88 

Not yet. It is possible to order that feature for 500 euro at https://wekan.team/commercial-support/

**Comment by belf88** [comment]
> @belf88
> 
> Not yet. It is possible to order that feature for 500 euro at https://wekan.team/commercial-support/

Release times?

**Comment by xet7** [comment]
@belf88 

Max 2 months after receiving payment. Maybe earlier.
